json/src/main/scala/generic/JSONMacros.scala

```diff
@@ -111,11 +111,12 @@ private[generic] object JSONMacros {
                   jsonProductApply(symbol.asClass)
                 )
               }
-            }.toVector
+            }.toList
```

Comment:
Because the BlockExtractor.apply expect a List: http://www.scala-lang.org/api/2.10.3/index.html#scala.reflect.api.Trees$BlockExtractor